src/drivers/bitbucket_cloud.js

```diff
@@ -253,10 +253,14 @@ class BitbucketCloud {
     });
 
     if (response.status > 300) {
-      const {
-        error: { message }
-      } = await response.json();
-      throw new Error(`${response.statusText} ${message}`);
+      try {
+        const {
+          error: { message }
+        } = await response.json();
+        throw new Error(`${response.statusText} ${message}`);
+      } catch (err) {
+        throw new Error(`${response.statusText}`);
+      }
```

Comment:
```suggestion
      try {
        const {
          error: { message }
        } = await response.json();
        throw new Error(message);
      } catch (err) {
        throw new Error(`${response.statusText} ${err.message}`);
      }
```

I was wondering if we could take advantage of the `catch` block to avoid repeating `response.statusText` in the error message. 🤔

Reply:
It would have worked on languages that use `return` and error checking instead of `throw` and error handling. 🙊